### PR TITLE
Link web install debian

### PR DESCRIPTION
**Pull request by jburel**
Point to installation instruction for web on debian 9

see https://trello.com/c/00CYQi2t/29-debian-installation-page

To test:

From debian installation page, check that the link for web installation points to the instructions for web on debian 9

### CODE REVIEW

File: omero/sysadmins/unix/install-web/walkthrough/omeroweb-install-debian-ice3.6.txt [hunk 1]
```diff
@@ -0,0 +1,179 @@
...
+    
+    zip=$(ls OMERO.py*.zip)
+    rm -f $zip
+    ln -s OMERO.py-* OMERO.py
+
+Install in the virtualenv created previously (run as root)::
+    
+    /home/omero/omerowebvenv/bin/pip install --upgrade -r /home/omero/OMERO.py/share/web/requirements-py27.txt
+
+Configure OMERO.web and generate nginx template (run as omero)::
```
Comment (kennethgillen) [inline omero/sysadmins/unix/install-web/walkthrough/omeroweb-install-debian-ice3.6.txt:59]: s/nginx/NGINX

File: omero/sysadmins/unix/server-debian9-ice36.txt [hunk 2]
```diff
@@ -111,7 +111,7 @@ Installing and running OMERO.web
 
 OMERO.web is deployed using Nginx, for more details on how to install
```
Comment (kennethgillen) [inline omero/sysadmins/unix/server-debian9-ice36.txt:112]: s/Nginx/NGINX

File: omero/sysadmins/unix/install-web.txt [hunk 3]
```diff
@@ -68,6 +68,7 @@ more specific walk-through listed below.
     install-web/walkthrough/omeroweb-install-centos7-ice3.6
     install-web/walkthrough/omeroweb-install-ubuntu-ice3.6
     install-web/walkthrough/omeroweb-install-osx-ice3.6
+    install-web/walkthrough/omeroweb-install-debian-ice3.6
```
Comment (kennethgillen) [inline omero/sysadmins/unix/install-web.txt:76]: Missing the `.. seealso::` entry to make this visible, line :62 

setting_up_omero_web_ _omero_documentation

**Review by kennethgillen — CHANGES_REQUESTED** [review]
If the "Nginx" are copy-paste from the other docs, perhaps we should leave for consistency, or open a PR to update everywhere.

The `install-web` .. seealso is the only important change afaik.

**Comment by jburel** [comment]
The replacement nginx=>NGINX will affect more files
I will do that in a follow-up PR

**Comment by kennethgillen** [comment]
👍 good to merge

**Comment by kennethgillen** [comment]
A little further research shows the NGINX web resources have inconsistent naming, so probably no need to be consistent ourselves.

screen shot 2017-08-02 at 17 31 10

**Pull request merged**